File: src/test/examples/adaptresourcefilenames/TestClass.java
// Copyright (c) 2018, the R8 project authors. Please see the AUTHORS file
// for details. All rights reserved. Use of this source code is governed by a
// BSD-style license that can be found in the LICENSE file.

package adaptresourcefilenames;

import adaptresourcefilenames.pkg.C;
import adaptresourcefilenames.pkg.innerpkg.D;

public class TestClass {

  public static void main(String[] args) {
    new B().method();
    new C().method();
    new D().method();
  }
}

File: src/test/java/com/android/tools/r8/rewrite/assertions/kotlintestclasses/TestClass.kt
// Copyright (c) 2020, the R8 project authors. Please see the AUTHORS file
// for details. All rights reserved. Use of this source code is governed by a
// BSD-style license that can be found in the LICENSE file.

package com.android.tools.r8.rewrite.assertions.kotlintestclasses

fun main() {
  try {
    Class1().m()
  } catch (e: AssertionError) {
    println("AssertionError in Class1")
  }
  try {
    Class2().m()
  } catch (e: AssertionError) {
    println("AssertionError in Class2")
  }
  println("DONE")
}